src/Types/Interface/Post.ts

```diff
@@ -0,0 +1,23 @@
+import { Channel } from './Channel';
+import { Comment } from './Comment';
+import { Like } from './Like';
+import { User } from './User';
+
+export interface Post {
+  likes: Like[];
+  comments: Comment[];
+  _id: string;
+  image?: string;
+  imagePublicId: string;
```

Comment:
![image](https://github.com/prgrms-fe-devcourse/FEDC5_STYLED_sehee/assets/70748442/1b46fd9c-b886-4746-8a2b-684167c52133)
`imagePublicId`도 선택 속성입니다!
```suggestion
  imagePublicId?: string;
```